src/components/common/OverviewItemCard.tsx

```diff
@@ -1,19 +1,13 @@
-import { Box, Theme, Typography, useTheme } from '@mui/material';
-import { makeStyles } from '@mui/styles';
-import { Icon, IconTooltip } from '@terraware/web-components';
+import { Box, Typography, useTheme } from '@mui/material';
+import { Button, Tooltip, IconTooltip } from '@terraware/web-components';
 import useDeviceInfo from '../../utils/useDeviceInfo';
-
-const useStyles = makeStyles((theme: Theme) => ({
-  editIcon: {
-    display: 'none',
-    fill: theme.palette.TwClrIcn,
-  },
-}));
+import strings from 'src/strings';
 
 type OverviewItemCardProps = {
   isEditable: boolean;
   hideEditIcon?: boolean;
   onClick?: () => void;
```

Comment:
Are we using the onClick somewhere in this component? If not, we could remove this prop